scripts/aem.js

```diff
@@ -541,29 +559,25 @@ async function loadBlock(block) {
     block.dataset.blockStatus = 'loading';
     const { blockName } = block.dataset;
     try {
-      const cssLoaded = loadCSS(`${window.hlx.codeBasePath}/blocks/${blockName}/${blockName}.css`);
```

Comment:
Extracted above in `loadModule` so we have a generic logic to use also to load the plugins